src/passa/cli/remove.py

```diff
@@ -37,30 +37,25 @@ def parsed_main(options):
     from passa.projects import Project
     from .lock import lock
 
-    packages = options.packages
-    # TODO: Ensure all lines are valid.
+    default = (options.only != "dev")
```

Comment:
FYI i hate the `only` semantic, I hope we can find a better name someday

Reply:
`passa remove black --in=dev` looks pretty neat to me too (but might be a little too cute)